### PR TITLE
Oxo patterns

### PR DESCRIPTION
**Pull request by Arens404**

### CODE REVIEW

File: oxoPattern1.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+            System.out.print("Line:");
+            int userInputLines = userInputScanner.nextInt();
+            System.out.print("Column:");
+            int userInputColumns = userInputScanner.nextInt();
+            for(int i = 1; i <= userInputLines; i++){
+                if(i % 2 == 0){
+                    for(int j = 1; j <= userInputColumns; j++){
+                        System.out.print("xo");
+                    }
+                } else {
```
Comment (Chezo) [inline oxoPattern1.java]: instead of 2 loops use variable set to change what to write and use 1 loop with this variable

File: oxoPattern2.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                    }
+                } else{
+                    if(i % 2 == 0){
+                        for(int j = 1; j <= userInputColumns; j++){
+                            System.out.print("xo");
+                            if(j % 2 == 0){
+                                System.out.print("II");
+                            }
+                        }
+
```
Comment (Chezo) [inline oxoPattern2.java]: see in oxopattern1

**Review by Chezo — CHANGES_REQUESTED** [review]

File: oxoPattern1.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class oxoPattern1 {
+    static Scanner userInputScanner = new Scanner(System.in);
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Select witch task u wanna try:");
+        try{
+            int UserInputContainer = scanner.nextInt();
```
Comment (Chezo) [inline oxoPattern1.java]: just an int

File: oxoPattern1.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            switch (UserInputContainer){
+                case 1 -> oxoPatternTaskOne();
+                case 2 -> oxoPatternTaskTwo();
+                default -> System.out.println("U have only two option! Please select from taskOne or taskTwo");
+            }
+        } catch (InputMismatchException IllegalArgumentException){
```
Comment (Chezo) [inline oxoPattern1.java]: mislead name-type pair

File: oxoPattern1.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+                for(int j = 1; j <= userInputColumns; j++){
+                    if(i % 2 == 0){
+                        System.out.print("ox");
+                    } else {
+                        System.out.print("xo");
+                    }
+                }
+            System.out.println();
+            }
+        } catch (InputMismatchException IllegalArgumentException) {
```
Comment (Chezo) [inline oxoPattern1.java]: mislead name-type pair

File: oxoPattern1.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+                            System.out.print("xo");
+                            if(j % 2 == 0){
+                                System.out.print("II");
+                            }
+                        }
+                    }
+                }
+                System.out.println();
+            }
+        } catch (InputMismatchException IllegalArgumentException) {
```
Comment (Chezo) [inline oxoPattern1.java]: mislead name-type pair

File: oxoPattern1.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class oxoPattern1 {
+    static Scanner userInputScanner = new Scanner(System.in);
```
Comment (Chezo) [inline oxoPattern1.java]: it could be final and must be private

**Review by Chezo — CHANGES_REQUESTED** [review]

File: oxoPattern1.java (hunk 7 above)
Comment (Arens404) [reply oxoPattern1.java]: 78fa8de2

File: oxoPatternTasksMain/Setter.java [hunk 8]
```diff
@@ -0,0 +1,10 @@
+package oxoPatternTasksMain;
+
+public class Setter {
+    private java.util.Scanner scanner = new java.util.Scanner(System.in);
+
+    public int input() {
+        int input = scanner.nextInt();
+        return input;
+    }
```
Comment (Chezo) [inline oxoPatternTasksMain/Setter.java]: dont really need it, why we created it?

**Review by Chezo — CHANGES_REQUESTED** [review]

**Review by Chezo — CHANGES_REQUESTED** [review]
please eliminate the extra class with scanner initialization.

File: oxoPatternTasksMain/Setter.java (hunk 8 above)
Comment (Arens404) [reply oxoPatternTasksMain/Setter.java]: bddedb09

**Comment by Arens404** [comment]
> please eliminate the extra class with scanner initialization.

bddedb09

**Review by Chezo — APPROVED** [review]

**Pull request merged**